src/main/java/Event.java

```diff
@@ -0,0 +1,36 @@
+public class Event extends Task {
+    String from;
```

Comment:
```suggestion
    protected String from;
```
perhaps you can consider making this variable protected to prevent accidental changes?